management/mail_log.py

```diff
@@ -679,7 +679,7 @@ def print_user_table(users, data=None, sub_data=None, activity=None, latest=None
                 data_accum[col] += d[row]
 
         try:
-            if None not in {latest, earliest}:
+            if None not in {tuple(latest), tuple(earliest)}:
```

Comment:
I don't think this works. For example, try:
```
if None not in {tuple([1]),tuple([2,None])}: print('yes') 
```
prints `yes` even though None is in the second array.